### DISCUSSION TITLE
Flutter Support

### DISCUSSION THREAD

**Discussion by JFOC** [zed-industries/zed#6532] [Feature Requests]
Would love to seeing Flutter support on Zed

**Comment by fdionisi** [comment]
Hey @JFOC, there's already an [open issue](https://github.com/zed-industries/community/issues/145) tracking Flutter support. Please, consider upvoting the original post. 😊

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
